include/psa/crypto_driver_contexts_composites.h

```diff
@@ -114,5 +114,17 @@ typedef union {
 #endif
 } psa_driver_aead_context_t;
 
+typedef union
+{
+    unsigned dummy; /* Make sure this union is always non-empty */
```

Comment:
The comment was copied as was the dummy member from the mac union above, but the reason appears to be a union with only one member breaks the initialisation code in crypto_struct.h - it will error with either braces round the initialisation value or not, for some reason (feels like a bit of a compiler bug, but a union with only one member is also a touch weird).